packages/run-protocol/src/econ-behaviors.js

```diff
@@ -397,3 +398,62 @@ export const configureVaultFactoryUI = async ({
   );
 };
 harden(configureVaultFactoryUI);
+
+/**
+ * Start the reward distributor.
+ *
+ * @param {BootstrapPowers & {
+ *   consume: { loadVat: ERef<VatLoader<DistributeFeesVat>>},
+ * }} powers
+ *
+ * @typedef {ERef<ReturnType<import('../../vats/src/vat-distributeFees.js').buildRootObject>>} DistributeFeesVat
+ */
+export const startRewardDistributor = async ({
+  consume: {
+    agoricNames,
+    chainTimerService,
+    bankManager,
+    loadVat,
+    vaultFactoryCreator,
+    ammCreatorFacet,
+    zoe,
+  },
+}) => {
+  const epochTimerService = await chainTimerService;
+  const distributorParams = {
+    epochInterval: 60n * 60n, // 1 hour
+  };
+  const [centralIssuer, centralBrand] = await Promise.all([
+    E(agoricNames).lookup('issuer', 'RUN'), // TODO: constant for RUN
```

Comment:
is this to do before merging or in the future? If the latter, it's best to have a ticket or a comment about who will do it.

Reply:
Good question. It's at least intended to start a conversation with reviewers.

Reply:
I think last time this came up we discussed a set of "well known names". `"RUN"` would be one of them. Seems like it wouldn't hurt to have a constant file now for what would eventually be well-known names, as long as we understand what the semantics are of a name being well-known. Though I can't say that I'm certain.